chips/nrf51/src/i2c.rs

```diff
@@ -0,0 +1,404 @@
+//! Implementation of I2C for nRF51.
+//!
+//! This module supports nRF51's two I2C master (`TWIM`) peripherals.
+
+extern crate nrf5x;
```

Comment:
This shouldn't be needed here, it is already imported in [nrf51/src/lib.rs](https://github.com/tock/tock/blob/master/chips/nrf51/src/lib.rs#L7)

Reply:
Doesn't compile without this.